ci/build.sh

```diff
@@ -41,12 +42,29 @@ cd ../..
 echo "Test docker image locally..."
 ruby deploy/test/test_docker.rb
 
-if [ -z "$DOCKER_PASSWORD" ] || [ -z "$TRAVIS_TAG" ]; then
-    echo "Skip Docker pushing"
-else
+if [ -n "$DOCKER_PASSWORD" ] && [ -n "$TRAVIS_TAG" ]; then
     echo "Pushing docker image with $DOCKER_TAG:$VERSION ..."
     echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
     docker push $DOCKER_TAG:$VERSION
+elif [ -n "$DOCKER_PASSWORD" ] && [ "$TRAVIS_BRANCH" == "master" ]; then
```

Comment:
TODO: 2nd CI still sees this as attempt to to push to master: https://travis-ci.org/SumoLogic/sumologic-kubernetes-collection/builds/564260193?utm_source=github_status&utm_medium=notification

Reply:
Also needs check "$TRAVIS_EVENT_TYPE" == "push"